cluster/kubemark/gce/config-default.sh

```diff
@@ -38,6 +38,7 @@ MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-cos-stable-60-9592-90-0}
 MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-cos-cloud}
 
 NETWORK=${KUBE_GCE_NETWORK:-e2e}
+SUBNETWORK="${SUBNETWORK:-${NETWORK}}"
```

Comment:
Doesn't setting this default kill `detect-subnetworks`?
https://github.com/kubernetes/kubernetes/blob/6a02660fbf11a7a84e2b4307958fd4fee857c585/cluster/gce/util.sh#L888-L910

Reply:
Thanks for looking, I'm just about to push another commit to avoid setting SUBNETWORK in config-default.sh.